src/systems/track_controller/TrackController.cc

```diff
@@ -268,6 +281,26 @@ void TrackController::Configure(const Entity &_entity,
   gzdbg << "Subscribed to " << corTopic << " for receiving track center "
          << "of rotation commands." << std::endl;
 
+  // Publish track odometry
+  const auto kDefaultOdometryTopic = topicPrefix + "/odometry";
+  const auto odometryTopic = validTopic({_sdf->Get<std::string>(
+    "odometry_topic", kDefaultOdometryTopic).first, kDefaultOdometryTopic});
```

Comment:
Document this parameter in the header.

Reply:
https://github.com/gazebosim/gz-sim/pull/2021/commits/ef405008ad8d92e98bca3cd1ea43127b8bdda3ee